registry/contract/src/lib.rs

```diff
@@ -259,12 +245,25 @@ impl Contract {
     pub fn register(
         &mut self,
         function_name: String,
+        forked_from: Option<IndexerIdentity>,
         code: String,
         schema: String,
         rule: Rule,
         start_block: StartBlock,
+        account_id: Option<String>,
```

Comment:
```suggestion
        account_id: Option<AccountId>,
```
If I recall correctly, inputs will be parsed to their defined types when the method is called. So setting `AccountId` makes the parsing below obsolete.